<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMulticastDemoServerClass</class>
 <widget class="QDialog" name="QMulticastDemoServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>组播发送</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>组播发送速度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSpeed"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>MB/s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MulticastDemoServer.qrc"/>
 </resources>
 <connections/>
</ui>
